boxsdk/object/recent_item.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+
+from __future__ import unicode_literals, absolute_import
+
+from .base_endpoint import BaseEndpoint
+from .base_api_json_object import BaseAPIJSONObject
+
+
+class RecentItem(BaseEndpoint, BaseAPIJSONObject):
+    """Represents a single recent item accessed by a Box user."""
+
+    _item_type = 'recent_item'
+
+    def __init__(self, session, response_object=None, **kwargs):
```

Comment:
We shouldn't accept `**kwargs` unless we're willing to do something with it.

Doing this is just a hack around the fact that we're passing `object_id` to it..

We should solve that problem a different way.

And if we remove `**kwargs`, then you can actually just remove this method override altogether, since it isn't doing anything other than calling `super`.